mk_verificator/utils/__init__.py

```diff
@@ -51,7 +51,35 @@ def get_groups(config):
             raise ValueError('Config file contains'
                              ' invalid groups name: {}'.format(invalid_groups))
 
-    return test_groups if test_groups else groups
+    groups = test_groups if test_groups else groups
+
+    # For splitting Ceph nodes
+    local_salt_client = init_salt_client()
+
+    if "ceph*" in groups:
+        groups.remove("ceph*")
+
+        # ToDo(den) Remake it !!!
+        ceph = [x.split('.')[0] for x in active_nodes if
+                ('ceph' in x.split('.')[0])]
+
+        ceph_status = local_salt_client.cmd(
+            'ceph-001*', "cmd.run", ["ceph -s"])
```

Comment:
Let's check ceph-mon service on all nodes instead of targeting some hardcoded minion name